mkdocs.yml

```diff
@@ -84,10 +84,12 @@ nav:
       - Tracing: observability/tracing.md
     - Persistence:
       - Responsibilities: persistence/responsibilities.md
-      - Buckets: persistence/buckets.md
+      - GCP storage alternatives:
```

Comment:
Burde vi sortert Aiven sine også i en egen kategori?

Reply:
Gjerne det for min del